plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -130,6 +133,20 @@ export const resolvers = ({
       _context = {}
     ): Observable<JobLink> {
       return runJob(args.id).map(({ jobId }) => ({ jobId }));
+    },
+    createJob(
+      _parent = {},
+      args: GeneralArgs,
+      _context = {}
+    ): Observable<JobLink> {
+      return createJob(args.data).map(({ jobId }) => ({ jobId }));
+    },
+    updateJob(
+      _parent = {},
+      args: GeneralArgs,
+      _context = {}
+    ): Observable<JobLink> {
+      return updateJob(args.id, args.data).map(({ jobId }) => ({ jobId }));
```

Comment:
on the JobModel-test.js there are some tests for the runJob, you could inspire on them to write some tests for those resolvers.